go/protocol/keybase1/teams.go

```diff
@@ -1608,19 +1612,21 @@ func (o TeamList) DeepCopy() TeamList {
 }
 
 type AnnotatedMemberInfo struct {
-	UserID         UID           `codec:"userID" json:"uid"`
-	TeamID         TeamID        `codec:"teamID" json:"team_id"`
-	Username       string        `codec:"username" json:"username"`
-	FullName       string        `codec:"fullName" json:"full_name"`
-	FqName         string        `codec:"fqName" json:"fq_name"`
-	IsImplicitTeam bool          `codec:"isImplicitTeam" json:"is_implicit_team"`
-	IsOpenTeam     bool          `codec:"isOpenTeam" json:"is_open_team"`
-	Role           TeamRole      `codec:"role" json:"role"`
-	Implicit       *ImplicitRole `codec:"implicit,omitempty" json:"implicit,omitempty"`
-	NeedsPUK       bool          `codec:"needsPUK" json:"needsPUK"`
-	MemberCount    int           `codec:"memberCount" json:"member_count"`
-	EldestSeqno    Seqno         `codec:"eldestSeqno" json:"member_eldest_seqno"`
-	Active         bool          `codec:"active" json:"active"`
+	UserID              UID           `codec:"userID" json:"uid"`
+	TeamID              TeamID        `codec:"teamID" json:"team_id"`
+	Username            string        `codec:"username" json:"username"`
+	FullName            string        `codec:"fullName" json:"full_name"`
+	FqName              string        `codec:"fqName" json:"fq_name"`
+	IsImplicitTeam      bool          `codec:"isImplicitTeam" json:"is_implicit_team"`
+	IsOpenTeam          bool          `codec:"isOpenTeam" json:"is_open_team"`
+	Role                TeamRole      `codec:"role" json:"role"`
+	Implicit            *ImplicitRole `codec:"implicit,omitempty" json:"implicit,omitempty"`
+	NeedsPUK            bool          `codec:"needsPUK" json:"needsPUK"`
+	MemberCount         int           `codec:"memberCount" json:"member_count"`
+	EldestSeqno         Seqno         `codec:"eldestSeqno" json:"member_eldest_seqno"`
+	Active              bool          `codec:"active" json:"active"`
+	AllowProfilePromote bool          `codec:"allowProfilePromote" json:"allow_profile_promote"`
+	IsMemberShowcased   bool          `codec:"isMemberShowcased" json:"is_member_showcased"`
```

Comment:
We should evaluate tech debt here. This struct is used in both team/list RPCs - when listing teams that we are members of, and also when listing all teammates. The bigger it grows the harder it is to understand which fields are used where.